File: C Programming (All Files)/C Programming (Lab Report 4)/Question_2.c
// Write a C Program To Define a Function In Directives.

#include <stdio.h>
#define SQUARE(x) ((x) * (x))

int main() 
{
    int num;
    int result;

    printf("Please Enter a Number To Know It's Square : ");
    scanf("%d", &num);

    result = SQUARE(num);
    
    printf("The Square Of The Given Number %d Is : %d", num, result);
    
    return 0;
}

File: Python Programming (All Files)/Experiment 2 (Conditional)/Question_2.py
# Write a Python Program To Check Whether a Given Number Is Multiple Of Five Or Not.

a = int(input("Please Enter a Number : "))
b = a // 5

if a % 5 == 0:
    print(f"{a} Is The {b} Multiple Of 5")
else:
    print(f"{a} Is Not a Multiple Of 5")